opentelekomcloud/resource_opentelekomcloud_lb_listener_v2.go

```diff
@@ -193,20 +227,23 @@ func resourceListenerV2Read(d *schema.ResourceData, meta interface{}) error {
 
 	log.Printf("[DEBUG] Retrieved listener %s: %#v", d.Id(), listener)
 
-	d.Set("id", listener.ID)
-	d.Set("name", listener.Name)
+	d.Set("region", GetRegion(d, config))
 	d.Set("protocol", listener.Protocol)
-	d.Set("tenant_id", listener.TenantID)
-	d.Set("description", listener.Description)
 	d.Set("protocol_port", listener.ProtocolPort)
-	d.Set("admin_state_up", listener.AdminStateUp)
+	d.Set("tenant_id", listener.TenantID)
+	d.Set("id", listener.ID)
```

Comment:
As far as I can see in other resources, ID should be set via `d.SetId(listener.ID)`

Reply:
I don´t get your point here, sorry.